pocs/utils/light_control.py

```diff
@@ -0,0 +1,141 @@
+#Basic Python interface for the qhue system.... ip for bridge -> "192.168.0.2"
+import qhue
+from os import path
+from qhue import Bridge, QhueException, create_new_username 
+
+#Setting Up Remote Access to Hue Bridge
+
+def ManualLogin(LED_State, Desk_State, ip):
+    
+    """
+    A way to manually login to the hue bridge, requires the button on the bridge to be pushed.
```

Comment:
Kind of unclear what the purpose of this is. Is it so someone can set the lights to a certain value using python? Why would auto login fail? Also, would it make sense to have 2 separate functions: one for setting up a manual login and one for setting the states? I can imagine the setting the states (given someone has already logged in) would be handy.